docs/source/model_design/vivarium_model_components/causes/index.rst

```diff
@@ -827,5 +827,43 @@ Morbidity Impacts
+
+In order to compute **years lived with disability (YLDs)** for a particular health outcome in a given population, the number of people living with that outcome is mulitplied by the DW for the health state associated with that sequela. Ultimately, YLDs are used to indicate burden of disease: DALYs are calculated as the sum of YLLs and YLDs. The DALY-based estimation of the burden of disease is important because it simultaneously considers the reduced health state due to disability before death and the decline in life expectancy due to death. It thus moves away from conventional measurements of the burden of disease that use readily available data on mortality, prevalence, and incidence (`Kim et al., 2022 <https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8841194/>`_). 
+
+YLD Uncertainty
+^^^^^^^^^^^^^^^
+
+The uncertainty ranges reported around YLDs incorporate uncertainty in prevalence and uncertainty in the DW. To do this, we take the 1000 samples of comorbidity-corrected YLDs and 1000 samples of the DW to generate 1000 samples of the YLD distribution. We assume no correlation in the uncertainty in prevalence and DWs. The 95% uncertainty interval is reported as the 25th and 75th values of the distribution. UIs for YLDs at different points in time (1990, 1995, 2000, 2005, 2010, and 2016) for a given disease or sequela are correlated because of the shared uncertainty in the DW. For this reason, changes in YLDs over time can be significant even if the UIs of the two estimates of YLDs largely overlap because significance is determined by the uncertainty around the prevalence estimates.
+
+
+Residual YLDs
+^^^^^^^^^^^^^
+
+For less common diseases and their sequelae, GBD may not currently estimate disease prevalence and YLDs, and have thus been included in residual categories. For these residual categories, we estimate YLDs by multiplying the residual YLL estimates by the ratio of YLDs to YLLs from the estimates Level 3 causes in the same disease category that were explicitly modelled. This scaling is done for each country-sex-year. 
+
+Incidence- vs. Prevalence-Based YLDs
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+*Incidence-based YLDs* provide a measure of the disease burden experienced by an individual over the course of their lifetime. Incidence-based YLDs are calculated by adding together YLD 1 (i.e., the product of the DW associated with disease 1 and the time between onset and survival) and YLD 2 (i.e., the product of the DW associated with disease 2 and the time between onset and survival). 
```

Comment:
I'm not sure I follow the example here (YLD 1 and YLD 2).

The key difference between the two, in my understanding, is that incidence-based YLDs immediately count all of the future disability caused by a disease at the time of incidence, while prevalence-based YLDs only count the disability at the time it is experienced. For example, an incident case of an incurable disease would only accrue one year of YLDs in the first year it is experienced if using prevalence-based YLDs, while it would accrue many years of YLDs if using incidence-based YLDs (as many years as are left in the person's life expectancy, because we know they cannot be cured).

Reply:
Yes, this aligns with my understanding as well! I can rephrase this part and take out the example of YLD 1 and 2. I wanted to give an example of how incidence-based YLDs can be the sum of all future disability caused by multiple diseases (rather than just representing the disability caused by a single disease) but I don't think I made that clear

Reply:
I like the example, but maybe separate it from the other parts of the text more? Clearly define the incidence vs prevalence YLDs first and then say if you have multiple diseases, you would calculate it by XYZ